src/main/java/uk/ac/cam/cl/dtg/isaac/dto/users/UserSummaryWithEmailAddressAndGenderDTO.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2017 James Sharkey
```

Comment:
I didn't know you travel back to 2017 to create this class and that you changed your name in the process. What's wrong with Kasim? It's a fine name :-P